modules/ROOT/pages/policies-landing-page.adoc

```diff
@@ -4,7 +4,46 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: policy, custom, ootb, offline
 
-A policy extends the functionality of an API and enforces certain capabilities such as security. A policy can control access and traffic without modification to the API implementation. Custom policies can provide other capabilities, such as encrypting and logging.
+Policies enable you to enforce regulations to help manage security, control traffic, and improve adaptability of your APIs. For example, a policy can control authentication, access, allotted consumption, and service level access (SLA). You can implement all these regulations with no modification to the code implementation. Mulesoft provides ready-to-use default policies that are shipped with the product. Additionally, you can also create custom policies based on your specific business requirements.
+
+You can apply policies to these types of APIs:
+
+* An APIkit project
++
+For example, deploy the APIkit project to Anypoint Platform using API Autodiscovery and apply a policy.
++
+* An API running on CloudHub
++
+Design an API on Anypoint Platform, configure a proxy for Cloudhub, and apply a policy.
+* An API deployed to a private or cloud-based Mule Runtime 3.8.x or later
++
+You can apply a policy to any API implemented with Mulesoft for any of the proxy types: RAML/Rest, HTTP, or SOAP-based as long as the proxy is exposed through an HTTP endpoint.
+You can also apply a policy to APIs not implemented with Mulesoft by deploying a Proxy application to control how and when a received request is forwarded to its implementation endpoint.
```

Comment:
We can add at the end of this sentence. API Manager supports the following proxy types: RAML/Rest, HTTP, or SOAP-based.

Reply:
Done! Thanks!

Reply:
Done! Thanks!